src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/template.json

```diff
@@ -518,7 +518,7 @@
       "type": "parameter",
       "datatype": "bool",
       "defaultValue": "false",
-      "displayName": "Do not use top-level statements",
+      "displayName": "Do not use _top-level statements",
```

Comment:
should there be a corresponding change in templatestrings.en.json as well, there is one for Components-WebAssembly

Reply:
😲 Updated the remaining templates here. 🤦🏽‍♀️